### PR TITLE
feat(settings): display username on logout hover

### PR DESCRIPTION
**Pull request by setchy**
Screenshot 2024-04-06 at 8 19 53 AM

### CODE REVIEW

File: src/routes/Settings.tsx [hunk 1]
```diff
@@ -230,10 +230,14 @@ export const SettingsRoute: React.FC = () => {
 
           <button
             className={footerButtonClass}
-            title="Logout"
+            title={`Logout ${accounts.user.login}`}
```
Comment (bmulholland) [inline src/routes/Settings.tsx:233]: How about "Logout of "?

Comment (bmulholland) [reply src/routes/Settings.tsx:233]: Logout from?

Comment (setchy) [reply src/routes/Settings.tsx:233]: Pretty even - https://trends.google.com/trends/explore?q=Logout%20of,Logout%20from&hl=en

Comment (setchy) [reply src/routes/Settings.tsx:233]: https://github.com/gitify-app/gitify/pull/983

**Pull request merged**